docs/source/tutorials/GEDIL4B/GEDI04_L4B documentation.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<p>This should be GEDI L4B I believe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>